### DISCUSSION TITLE
Cleanup button that reorganises items in a clean organised fashion

### DISCUSSION THREAD

**Discussion by pbiggar** [darklang/dark#4363] [Feedback]
A user says:
>Some things I've written down while exploring:
>- Adding a link to create a new canvas
>    - popup with inputbox → submit → redirect to new canvas
>- Adding a link on a canvas to a specific homepage for static assets
>- Add a docs link in the sidebar
>
>Canvas
>- Don't create new items behind existing items
>- Zooming on canvas
>- Draw rectangle to select elements and drag multiple
>- Cleanup button that reorganises items in a clean organised fashion
>
>Datastores
>- Showing key explicit but readonly in list of database properties (It was confusing to me if I should add a key myself, or if it was there automatically)
>- Reference database key in other database
instead of having a people table with pets: [String] having a people table with pets:
>    - [Pets::key]

**Comment by StachuDotNet** [comment]
> Adding a link on a canvas to a specific homepage for static assets

I'm not totally sure what this means.
